source/util/AMDSErrorMonitor.h

```diff
@@ -0,0 +1,275 @@
+/*
```

Comment:
But why? Is there something specific that doesn't allow you to just include AMErrorMonitor?